fill/fill_station.cc

```diff
@@ -131,12 +139,30 @@ class TelemeterServiceImpl final : public FillStationTelemeter::Service
     }
 };
 
+class RocketTelemeterServiceImpl final : public RocketTelemeter::Service 
+{
+    Status StreamTelemetry(ServerContext *context, const RocketTelemetryRequest *request,
+                       ServerWriter<RocketTelemetry> *writer) override
+    {
+        while (true) {
+            auto now = std::chrono::high_resolution_clock::now();
+            RocketTelemetry t = protoBuild.buildProto();
+            if (!writer->Write(t)) {
+                // Broken stream
+                return Status::CANCELLED; 
+            }
+            std::this_thread::sleep_until(now + std::chrono::milliseconds(1000));
```

Comment:
I don't think we should sleep here since we want to read rocket telemetry as soon as it comes in